WORKSPACE

```diff
@@ -45,6 +45,47 @@ new_http_archive(
   sha256 = "3d374a730aef330424a356a8d4f04d8a54277c425e274ecb7d9c83aa912c6bfa"
 )
 
+# Dependencies to support TensorBoard functional tests.
+http_archive(
+    name = "io_bazel_rules_go",
+    sha256 = "ef1aa6a368808d3aa18cbe588924f15fb8fac75d80860080355595e75eb9a529",
+    strip_prefix = "rules_go-0.4.0",
+    url = "https://github.com/bazelbuild/rules_go/archive/0.4.0.tar.gz",
```

Comment:
Please change url to `urls = [...]` and add http://bazel-mirror.storage.googleapis.com/github.com/bazelbuild/rules_go/archive/0.4.0.tar.gz to the beginning of the list.